ui/e2e/events/history.spec.ts

```diff
@@ -120,6 +120,27 @@ test("it renders, filters, and paginates events", async ({ page }) => {
   await paginationWrapper.getByRole("button", { name: "3" }).click();
   await expect(page.getByTestId("event-row")).toHaveCount(2);
 
+  /**
+   * Test the select options for pagesize
+   */
+
+  const selectPagesize = page.getByTestId("select-pagesize");
```

Comment:
When you do `const selectPagesize = page.getByRole("combobox");` you can even remove the test id